precompiles/types/coin.go

```diff
@@ -1,25 +1,25 @@
-package bank
+package types
 
 import (
 	"math/big"
 
 	"cosmossdk.io/math"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ethereum/go-ethereum/common"
-
-	ptypes "github.com/zeta-chain/node/precompiles/types"
 )
 
+const ZRC20DenomPrefix = "zrc20/"
+
 // ZRC20ToCosmosDenom returns the cosmos coin address for a given ZRC20 address.
 // This is converted to "zevm/{ZRC20Address}".
 func ZRC20ToCosmosDenom(ZRC20Address common.Address) string {
 	return ZRC20DenomPrefix + ZRC20Address.String()
 }
```

Comment:
We can move this to cmd/zetacored/config where we maintain the other prefixes

Reply:
Applied in 06d0c5d5a8a2d7ce54f84a666c7bef7784963ccf